plugins/woocommerce/src/Internal/Admin/ProductReviews/ReviewsListTable.php

```diff
@@ -569,6 +569,17 @@ protected function handle_row_actions( $item, $column_name, $primary ) : string
 			);
 		}
 
+		/**
+		 * Filters the action links displayed for each review in the Reviews list table.
+		 *
```

Comment:
Lint error: `Docblock comment was found for the hook but does not contain a "\@since" versioning.PHP_CodeSniffer(WooCommerce.Commenting.CommentHooks.MissingSinceComment)`


```suggestion
		/**
		 * Filters the action links displayed for each review in the Reviews list table.
		 *
		 * @since 9.8.0
```

Reply:
Thanks - I didn't know what version you all wanted to put there.